### ISSUE TITLE
@mediapipe/tasks-vision fails to load vision_wasm in combination with vue3

### ISSUE THREAD

**Issue by jonkad** [google/mediapipe#4827]
### Have I written custom code (as opposed to using a stock example script provided in MediaPipe)

Yes

### OS Platform and Distribution

Web / vue 3.3.4 / vite 4.4.9

### Programming Language and version

Javascript / Typescript

### Solution

tasks-vision

### Describe the actual behavior

failed to load files for vision tasks

### Describe the expected behaviour

Should load files as expected

### Standalone code/steps you may have used to try to get what you need

```shell
Test script: https://github.com/jonkad/vue3-livekit-tasks-vision
```

### Other info / Complete Logs

```shell
Error message in console: "Jsloader error (code #0): Error while loading script https%3A%2F%2Fcdn.jsdelivr.net%2Fnpm%2F%40mediapipe%2Ftasks-vision%400.10.5%2Fwasm/vision_wasm_internal.js"
```

**Comment by paulorenanmelo** [comment]
I am using 0.10.6 with a typescript based off the example [here](https://codepen.io/mediapipe-preview/pen/gOKBGPN), converted to javascript using tsc index.ts, and bundled with webpack. When I use http-server to serve the dist folder I get 
`"GET /@mediapipe/tasks-vision/wasm/vision_wasm_internal.js" Error (404): "Not found"`
Any idea how to solve this?

**Comment by MaskeZen** [comment]
Hello @schmidt-sebastian , I have the same error that @jonkad, when I build the solution the request is made to this filename:

`/https%3A%2F%2Fcdn.jsdelivr.net%2Fnpm%2F%40mediapipe%2Ftasks-vision%400.10.0%2Fwasm/vision_wasm_internal.js`

I see that the bars are replaced by the UTF-8 character code, with your first comment I thinked that this will be fixed in the version 0.10.6, but the last comment confused me a little.

If I try to use a local location to the file I have the same problem if the files are not in the root directory, for example:

`/%2Fassets%2Fmediapipe%2Fwasm/vision_wasm_internal.js`

this is the problem of that bug or I am having other issue here?

Thanks for you help!

**Comment by schmidt-sebastian** [comment]
Thanks for the update. This was the fix we released in 0.10.6.

**Comment by VivianManohar** [comment]
Hi @schmidt-sebastian I face similar issue in Vue 3. I am using the latest tasks-vision though. As soon as I open the page, it freezes without any console errors. Then I can't even use the back button in the browser. Only way to quit is to close the tab. It happens in both Chrome and Safari.

Here's how I try to load the hand_landmarker model:

In package.json:  

"@mediapipe/tasks-vision": "^0.10.12"

In my .vue page:

import { HandLandmarker, FilesetResolver } from '@mediapipe/tasks-vision';

const createHandLandmarker = async () => {
  const vision = await FilesetResolver.forVisionTasks("https://cdn.jsdelivr.net/npm/@mediapipe/tasks-vision@0.10.12/wasm"
);
  handLandmarker = await HandLandmarker.createFromOptions(vision, {
    baseOptions: {
      modelAssetPath: "/tensorflow_models/hand_landmarker.task",
      delegate: "GPU",
    },
    runningMode: "VIDEO",
    numHands: 2,
  });
};

The network tab shows that the following are fetched:
@mediapipe_tasks-vision.js?v=d7025b03 | 200
vision_wasm_internal.js | 200 
vision_wasm_internal.wasm | 200 

I tried different versions as well, with the same result. And FYI, I use Vite in my Quasar project. As a newbie, it would be great if I could get some help with this. Many thanks!
